src/test/ts/browser/PluginTest.ts

```diff
@@ -1,17 +1,202 @@
-import { TinyAssertions, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from "@ephox/mcagar";
+import * as TestUtils from '../module/Utils';
 
 import Plugin from '../../../main/ts/Plugin';
 
 // This an example of a browser test of the editor.
-describe('browser.PluginTest', () => {
+describe('browser.tiynmce.plugin.enhancedlist.PluginTest', () => {
   const hook = TinyHooks.bddSetup({
-    plugins: 'shan-enhanced-lists',
-    toolbar: 'shan-enhanced-lists'
-  }, [ Plugin ]);
+    plugins: 'shan-enhanced-lists lists advlist',
+    toolbar: 'shan-enhanced-lists enhanced-list-ordered-list enhanced-list-unordered-list'
+  }, [Plugin]);
 
-  it('test click on button', () => {
+  const listStyleTest = (title: string, action: (editor) => Promise<void>, definition: Record<string, string>) => {
```

Comment:
```suggestion
// Move this interface to the top of the test file
interface Definition {
  readonly inputContent: string;
  readonly type: string;
  readonly style: string;
  readonly expected: string;
}
 
  const listStyleTest = (title: string, action: (editor) => Promise<void>, definition: Definition) => {
```
Nice to put this as an interface, don't ask me why but it's standard in tinymce.

Reply:
Yeah. Always make the types as specific as you can. You want the type-checker to help you identify typos or refactoring changes, and that's not very easy with generic things like `Record<string, string>`